src/core/particle.cpp

```diff
@@ -909,12 +909,16 @@ float Particle::ReturnLogLikelihood(Image &input_image, CTF &input_ctf, Reconstr
 //	particle_image->QuickAndDirtyWriteSlice("diff.mrc", 1);
 	// This low-pass filter reduces the number of independent pixels. It should therefore be applied only to
 	// the reference (temp_projection), and not to the difference (temp_particle - temp_projection), as is done here...
-	if (classification_resolution_limit > 0.0)
+//	temp_particle->ForwardFFT();
+//	if (classification_resolution_limit < 20.0f) temp_particle->CosineMask(original_pixel_size / 20.0f, original_pixel_size / 10.0f, true);
+	if (classification_resolution_limit > 0.0f)
 	{
 		temp_particle->ForwardFFT();
 		temp_particle->CosineMask(original_pixel_size / classification_resolution_limit, original_pixel_size / mask_falloff);
+//		temp_particle->CosineMask(0.75f * original_pixel_size / classification_resolution_limit, original_pixel_size / classification_resolution_limit);
```

Comment:
Do we know the justification for this / does it actually improve things?

Reply:
@ngrigorieff any ideas?

Reply:
Are you asking if the commented out code, which is new, does anything? I am not clear on the question